### ISSUE TITLE
Unexpected large minting

### ISSUE THREAD

**Issue by scottyeager** [threefoldtech/minting_v3#11]
In Septembers minting event, on farmer noticed two transactions from the minting wallet much larger than anything they would expect for their nodes:

Operation: [182950535770202113](https://stellar.expert/explorer/public/tx/182950535770202112#182950535770202113) 
Amount: 61,570.9875 TFT
Receiving wallet: GDGTVMSSE6S6Q3PN6KF4Q4MLLDV4WWNLJM5D34V5DQ75HCMLXGBQNO4Y
Memo (Hex HASH): fc8c173c46e7752f9d13f635335e2cf643aa75214baefccc9dc919660c6b9b6d

Operation: [182950522885488641](https://stellar.expert/explorer/public/op/182950522885488641)
Amount: 10,140.6125 TFT
Receiving wallet: GDGTVMSSE6S6Q3PN6KF4Q4MLLDV4WWNLJM5D34V5DQ75HCMLXGBQNO4Y
Memo (Hex HASH): 6dba4071b6251833301cdf74fc3d026045643a257743715fc7e96862d1624625

The attached hashes don't return any results from the minting receipt visualizer. This farmer operates nodes on testnet, and these transactions seem to be linked to two testnet nodes with gateway configurations (nodes 219 and 222).

**Comment by LeeSmet** [comment]
Yes, this was checked yesterday morning. The 2 payouts are related to test net nodes on v3. The receipt visualizer currently does not support test net receipts.

Regarding the actual payout, this is correct as far as the minting is concerned. With the introduction of gold farming, the minting code was updated to use farming policies from the chain itself, rather than hard coded values. As it turns out, on test net, there is a farming policy with ridiculous rewards for IP usage and NU. Iirc, values in that policy are something north of 500 USD per hour of public IP usage and more than 2 USD per NU (GB of public traffic). These 2 nodes had 10 hours of IP usage combined and about 2NU, hence the huge payout.

TL;DR: There is a broken farming certification on testnet which needs to be removed and unlinked from these nodes, or needs to be adjusted. The minting did what is was supposed to do.
